package.json

```diff
@@ -38,15 +41,14 @@
     "tasks/",
     "test/",
     "README.md",
-    "package-lock.json",
-    "yarn.lock"
+    "package-lock.json"
   ],
   "scripts": {
     "lint": "eslint \"./tasks/**/*.js\" \"./test/**/*.js\"",
     "gh-pages": "rimraf ./docs && cd ./template && npm run build && cd ../ && cp -rv ./template/public/ ./docs",
     "test:template": "cd template && npm i && export NODE_ENV=production && npm run test && npm run lint && npm run build",
     "test:template-backup": "export NODE_ENV=production && npm i --prefix template && npm test --prefix template && npm run lint --prefix template && npm run build --prefix template",
-    "test": "mocha --timeout 2000"
+    "test": "mocha --timeout 5000"
```

Comment:
I kept experiencing timeout issues locally on the last test, so increased this a bit more.  I think it's fine since there's a lot of filesystem manipulation happening which can take a lot of time to happen, depending on the machine running the tests.